### DISCUSSION TITLE
[Feature] Bring back (optional) screen auto dimming

### DISCUSSION THREAD

**Discussion by raisinbear** [immich-app/immich#14791] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hey,

as always thank you guys so much for Immich and all the hard work you are doing!

I noticed mobile screen auto dimming was removed in
#14699 . Apparently, it caused some issues (that I luckily didn’t encounter). I thought it was one of the very important progressions giving background upload limitations, at least on iOS.

Usually uploading is not a big deal. On home network, it is blazing fast and most of the time quickly done. But when on holiday, for instance, things look anlog different:
- I at least take a lot more pictures
- upload speed is often extremely slow. 

Example: me and my partner went on a holiday just recently. Of course we set up a shared album. We took _a lot_ of pictures daily that we would upload over the 10 MBit uplink of our Airbnb there at the end of the day. This would at times take far more than an hour. So dimming the screen was a very welcome feature, at least partially reducing power consumption and heat and chances of burn in.

It would be great to see that feature back, at least as an „experimental“ opt-in sort of option!

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by zackpollard** [comment]
We decided to remove this feature. The wakelock is still there so the screen won't turn off, but you can just manually turn down your screen brightness to minimum.

**Reply by raisinbear** [reply]
> We decided to remove this feature. The wakelock is still there so the screen won't turn off, but you can just manually turn down your screen brightness to minimum.

Yeah, I read about issues it was causing and understand it wasn’t deemed worthy to fully maintain.

My personal experience was flawless and I found it very practical. Also, manually dimming messes up Auto brightness on iOS and it doesn’t light back up when done, obviously. So it’s more of a ponder-if-it-takes-long-enough-that-I-should-care, manually dim screen, go to settings to turn off and on auto brightness instead of just letting it do it’s thing.
Hence the „opt-in“ suggestion. 🙃
